src/test/java/rest_api/petshop_test/PetTest.java

```diff
@@ -0,0 +1,173 @@
+package rest_api.petshop_test;
+
+import io.restassured.RestAssured;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.http.ContentType;
+import io.restassured.path.json.JsonPath;;
+import io.restassured.specification.RequestSpecification;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;;
+import rest_api.petshop.OrderStatus;
+import rest_api.petshop.PetStatus;
+import rest_api.petshop.dto.OrderDto;
+import rest_api.petshop.dto.PetDto;
+import org.testng.annotations.Test;
+import rest_api.petshop.dto.UserDto;
+
+import java.util.List;
+
+import static io.restassured.RestAssured.*;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class PetTest {
+    private final static String BASE_URI = "https://petstore.swagger.io/v2";
+    private RequestSpecification requestSpecification;
+    private PetDto pet;
+    private UserDto user;
+    private OrderDto order;
+
+    @BeforeClass
+    public void configureRestAssure() {
```

Comment:
Please, move all this configuration to some `TestBase `class

And create some `unconfigureRestAssured()` method where all settings should be reset.

Reply:
Done